doc/conf.py

```diff
@@ -166,6 +167,9 @@
 # If false, no module index is generated.
 #latex_use_modindex = True
 
+PY_VERSION = 3 if bson.py3compat.PY3 else 2
+
 intersphinx_mapping = {
     'gevent': ('http://www.gevent.org/', None),
+    'py': ('https://docs.python.org/%i/' % (PY_VERSION,), None),
```

Comment:
@ShaneHarvey @behackett should we link to Py2 or Py3 standard library documentation? I feel like this isn't a great solution since we only ever have a single version of the docs hosted so we'd link to whatever Python version is used to compile the docs. Personally, I think we should just link to Python 3 docs.

Reply:
I think we should always link to Python 3.